src/atoms/Link/Link.native.js

```diff
@@ -0,0 +1,71 @@
+import React, { useContext, useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import styled, { ThemeContext } from 'styled-components/native';
```

Comment:
You can probably do [this](https://styled-components.com/docs/basics#simpler-usage-with-the-metro-bundler)

Reply:
https://github.com/styled-components/styled-components/issues/2536

Running into this after trying that. The workaround mentioned doesn't seem clean


Would it be okay if we did this?

**.babelrc.js:**
```
plugins: [
        [
          'module-resolver',
          {
            alias: {
              '@storybook/react': '@storybook/react-native',
              'styled-components': 'styled-components/native',
            },
          },
        ],
      ],
```

Reply:
@kamleshchandnani need feedback on this

Reply:
looks good to me. But might break on consumers if they don't add the same alias. Let it be explicit

Reply:
Hmm yeah